webpack.config.ts

```diff
@@ -109,4 +109,14 @@ export const mainConfig: Configuration = {
 	resolve: {
 		extensions: ['.js', '.ts', '.jsx', '.tsx', '.css', '.json'],
 	},
+	plugins: [
+		new DefinePlugin({
```

Comment:
The same exact definitions occur for the renderer process, I think it’s worth defining the vars once in module-global scope and just passing them as plugins for each process